packages/yoroi-extension/app/ergo-connector/components/signin/SignTxPage.scss

```diff
@@ -1,8 +1,10 @@
 .component {
-  padding: 30px 15px 50px 15px ;
+  height: calc(100vh - 92px);
```

Comment:
The header is 52px (in `app/ergo-connector/components/layout/Layout.scss`), why deduce 92px ?

Reply:
On nightly/testnet, there is this badge that will be shown on top of the page. if I just add `52px` and the badge is where the buttons will be hidden out of the page. and this is the main point of the PR 😄. 

Check the image blow. This is how it looks like with `52px` with the testnet badge
<img width="635" height="635" alt="Screen Shot 2022-02-23 at 9 35 28 AM" src="https://user-images.githubusercontent.com/72753578/155277743-6a563e07-75af-4e56-81e9-2b4ff72740be.png">

Reply:
Ah, right. I forgot the banner.